dockerfiles/tasks.py

```diff
@@ -146,12 +157,35 @@ def pull(c, only_latest=False):
 def test(c, arguments='', running=True):
     """Run all test suite using ``tox``."""
     if running:
-        c.run(f'{DOCKER_COMPOSE_COMMAND} exec -e GITHUB_TOKEN=$GITHUB_TOKEN web tox {arguments}', pty=True)
+        c.run(
+            f'{DOCKER_COMPOSE_COMMAND} exec -e GITHUB_TOKEN=$GITHUB_TOKEN web tox {arguments}', pty=True)
     else:
         c.run(f'{DOCKER_COMPOSE_COMMAND} run -e GITHUB_TOKEN=$GITHUB_TOKEN --rm --no-deps web tox {arguments}', pty=True)
 
+
 @task
 def buildassets(c):
     """Build all assets for the application and push them to backend storage"""
     c.run(f'docker-compose -f {DOCKER_COMPOSE_ASSETS} run --rm assets bash -c "npm ci && node_modules/bower/bin/bower --allow-root update && npm run build"', pty=True)
     c.run(f'{DOCKER_COMPOSE_COMMAND} run --rm web python3 manage.py collectstatic --noinput', pty=True)
+
+
+@task(help={
+    'file': 'SQL File that should be use to go back old state',
+    'running': 'Execute in a running container',
+})
+def restoredb(c, file, running=True):
+    """Restore to old db state when running --backupdb migrate"""
+    subcmd = 'run --rm'
+    if running:
+        subcmd = 'exec'
+
+    c.run(f"{DOCKER_COMPOSE_COMMAND} 'start database'", pty=True)
+
+    c.run(f"docker cp {file} community_database_1:/tmp/dump.sql")
```

Comment:
Here you need to use the same trick we use in https://github.com/readthedocs/common/pull/98/files#diff-302a2219ef42d627f0daf3e4752a1838c41e64b43d7912f149f10cd04f6c30c0R109 because `community_` is only for our community code, but for corporate code (readthedocs.com) that should be `commercial_` instead.

So, you need to grab that value from the context.

Reply:
@vitorsv1 would you like to update this PR with this change requested?

Reply:
I think it would be possible to use `{DOCKER_COMPOSE_PROJECT_NAME}_database_1` in this string following #145